packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -368,8 +369,26 @@ const curationEmailDelay = new EventDebouncer({
   }
 });
 
+getCollectionHooks("Posts").editAsync.add(async function CuratedAuthorsNotification(post: DbPost, oldPost: DbPost) {
```

Comment:
Given there is now an EAF version of this and a LW/AF version, could you give the each of the functions a name that indicates this?

Reply:
Sure, but to clarify, the emails are different - ours goes to the post authors, while LW/AF goes to other site users (who are subscribed to get curated posts in their email)